src/test/java/edu/unc/lib/boxc/migration/cdm/SourceFilesCommandIT.java

```diff
@@ -573,6 +573,30 @@ public void statusUnmappedVerboseTest() throws Exception {
         assertOutputMatches(".*Potential Matches: +0.*");
     }
 
+    @Test
+    public void statusUnmappedPdfTest() throws Exception {
+        testHelper.indexExportData("pdf");
+
+        String[] args = new String[] {
+                "-w", project.getProjectPath().toString(),
+                "source_files", "generate",
+                "-b", basePath.toString()};
+        executeExpectSuccess(args);
+
+        String[] args2 = new String[] {
+                "-w", project.getProjectPath().toString(),
+                "source_files", "status",
+                "-v" };
+        executeExpectSuccess(args2);
+
+        assertOutputMatches(".*Last Updated: +[0-9\\-T:]+.*");
+        assertOutputMatches(".*Previous Files Mapped: +0.*");
```

Comment:
Should it be 0 across the board? I know all the file objects get dropped, but I guess it doesn't detect the pdf object itself as a file that needs to be mapped?

Reply:
I forgot to add `ENTRY_TYPE_DOCUMENT_PDF` to the `StatusQueryService.countIndexedFileObjects` query. Now it should say there's one file in the project and one unmapped object.